packages/frontend/src/pages/project/common/getProjectHeader.ts

```diff
@@ -99,8 +95,7 @@ export function getProjectHeader(
     purposes: project.display.purposes,
     technology: project.display.category,
     tvlBreakdown: project.config.escrows.length > 0 ? tvlBreakdown : undefined,
-    showTvlBreakdown:
-      config.features.tvlBreakdown && !!apiProject && !!tvlBreakdownProject,
+    showTvlBreakdown: config.features.tvlBreakdown && !!apiProject,
```

Comment:
So this will cause a bug now, as we will display breakdown for all projects, regardless if they have one?